ClientKit/Display.cs

```diff
@@ -26,7 +26,9 @@
 using ViewerCount = System.UInt32;
 using EyeCount = System.Byte;
 using SurfaceCount = System.UInt32;
-using DistortionPriority = System.Int32;
+using DistortionPriority = System.Int32;
+using DisplayInputCount = System.Byte;

```

Comment:
.Net has a signed ```byte``` (unsigned int8) and ```sbyte``` (signed int8), which are C# keyword aliases for the ```System.Byte``` and ```System.SByte``` types, respectively. I think it's Java that doesn't have a built-in unsigned 8-bit primitive.